src/Widgets/LinkContainerWidget/LinkContainerWidget.scss

```
@@ -0,0 +1,74 @@
+@import "src/assets/stylesheets/variables";
+
+.link-container-widget {
+  list-style-type: none;
+  padding: 0;
+  display: block;
+  color: $theme-greydark;
+
+  li {
+    display: block;
+    color: $theme-greydark;
+
+    a {
+      display: block;
+      color: $theme-greydark;
+    }
+
+    span {
+      display: block;
+      color: $theme-greydark;
```

I would move this part out of `.link-container-widget` and instead write this:

```scss
.link-container-widget,
.link-container-widget li,
.link-container-widget li a,
.link-container-widget li span {
  display: block;
  color: $theme-greydark;
}
```

---

or maybe like this inside `.link-container-widget`?

```suggestion
  &,
  & li,
  & li a,
  & li span {
    display: block;
    color: $theme-greydark;
  }
```

---

If that works, great!

---

If that works, great!